src/datatypes2/src/vectors/operations/find_unique.rs

```diff
@@ -60,20 +61,20 @@ pub(crate) fn find_unique_scalar<'a, T: ScalarVector>(
     }
 }
 
-// pub(crate) fn find_unique_null(
-//     vector: &NullVector,
-//     selected: &mut BitVec,
-//     prev_vector: Option<&NullVector>,
-// ) {
-//     if vector.is_empty() {
-//         return;
-//     }
+pub(crate) fn find_unique_null(
+    vector: &NullVector,
+    selected: &mut BitVec,
+    prev_vector: Option<&NullVector>,
+) {
+    if vector.is_empty() {
+        return;
```

Comment:
Should we set all position of `selected` to `false` here ?

Reply:
selected should also be empty